### PR TITLE
[Spec] MSC2285 Stable/Unstable crossover period support

### PR DESCRIPTION
**Pull request by EdGeraghty**
### Types
- [ ] Fix (non-breaking change which fixes an issue)
- [ ] Feature (non-breaking change which adds functionality)
- [ ] Refactor (non-breaking change which improves code quality - QA thoroughly )
- [ ] Breaking Change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Chore (updates that would not affect or change the code involving the app itself)

### QA

- [ ] Signup
- [ ] Login
- [ ] Logout
- [ ] Unencrypted Chat 
- [ ] Encrypted Chat 
- [ ] Group Chats
- [ ] Profile Changes
- [ ] Theming Changes
- [ ] Force Kill and Restart

### Final Checklist
 
- [x] All associated issues have been linked to PR
- [ ] All necessary changes made to the documentation
- [ ] Parties interested in these changes have been notified
- [x] Linter has been run on all code in the PR

### CODE REVIEW

File: lib/store/settings/theme-settings/selectors.dart [hunk 1]
```diff
@@ -220,7 +221,19 @@ double? selectAppBarElevation(ThemeType themeTypeNew) {
...
+    case ReadReceiptTypes.Off:
+      return Strings.labelOff;
+
+    case ReadReceiptTypes.On:
+      return Strings.labelOn;
+
+    case ReadReceiptTypes.Private:
+      return Strings.labelPrivate;
+
+    default: //I've not been coded for this one yet
```
Comment (ereio) [inline lib/store/settings/theme-settings/selectors.dart:234]: What is the functionality expected behind this one?

**Review by ereio — COMMENTED** [review]
Other than the question I posted, this looks good. Great work! 

Once that's clear, will approve. Can we set the switch block default to return `Strings.labelOn`? I'm assuming the actual default setting will still be "Off" correct?

File: lib/store/settings/theme-settings/selectors.dart (hunk 1 above)
Comment (EdGeraghty) [reply lib/store/settings/theme-settings/selectors.dart:234]: Purely defensive. The only two in the spec are handled above it. Feel free to patch out as wanted, it's on origin

Comment (EdGeraghty) [reply lib/store/settings/theme-settings/selectors.dart:234]: Note this is purely to handle string translation

**Review by ereio — APPROVED** [review]
Great work!

**Pull request merged**